superset-frontend/src/SqlLab/components/ShareSqlLabQuery/index.tsx

```diff
@@ -76,10 +77,10 @@ const ShareSqlLabQuery = ({
   const getCopyUrlForSavedQuery = (callback: Function) => {
     let savedQueryToastContent;
 
-    if (remoteId) {
+    if (uuid) {
       savedQueryToastContent = `${
         window.location.origin + window.location.pathname
-      }?savedQueryId=${remoteId}`;
+      }?savedQueryId=${uuid}`;
```

Comment:
Is there a concern that by switching to UUID existing URLs will break?

Reply:
Existing URLs will still works just fine as the get saved query endpoint still accepts numerical id's